src/components/AttentionBox/AttentionBoxLink/AttentionBoxLink.tsx

```diff
@@ -1,15 +1,19 @@
 import { FC } from "react";
 import styles from "./AttentionBoxLink.module.scss";
+import Link, { LinkProps } from "../../Link/Link";
+import cx from "classnames";
 
-interface AttentionBoxLinkProps {
-  href: string;
-  text: string;
-}
-const AttentionBoxLink: FC<AttentionBoxLinkProps> = ({ href, text }) => {
+const AttentionBoxLink: FC<LinkProps> = ({
+  href,
+  text,
+  // TODO: use Link's target default in next major
+  // For backward compatibility - using _self as default
+  target = Link.target.SELF,
+  className,
+  ...linkProps
```

Comment:
make sure nothing is required btw
as it would be breaking

Reply:
Yes thanks I checked, all Link props are optional